Integrations/integration-IsItPhishing.yml

```diff
@@ -0,0 +1,166 @@
+        var body = {
+            name: params.name,
+            license: params.license,
+            version: '2',
+            force: force,
+            url: url,
+            area: area,
+            timeout: timeout
+        };
+        if (!area) {
+            delete body.area;
+        }
+        if (!timeout) {
+            delete body.timeout;
+        }
+        var res = sendRequest('POST', 'check', body);
+        var resBody = res.Body.trim();
+        switch (resBody){
+            case 'SPAM':
+                return 'URL was identified as spam.';
+            case 'PHISHING':
+                return 'URL was identified as phishing.';
+            case 'UNKNOWN':
+                return 'URL is clean.';
+            case 'TIMEOUT':
+                return 'Timeout for the request has been reached. No verdict was returned for the request, and the URL should be considered clean.';
+            case 'NOT_EXPLORED':
+                return 'The URL was not analyzed as triggering the analysis may cause collateral damage (unsubscribe, order conformation, etc.)';
+            case 'NOT_AUTHORIZED':
+                return 'Authorization has failed for one of the following reasons:\n• Invalid customer name,\n• Invalid customer license.';
+            case 'REVOKED':
+                return 'The license provided is no longer valid for one of the following reasons:\n• Validity period has expired,\n• License has been revoked.';
+        }
+        if (res.substring(0,17) == 'TOO_MANY_REQUESTS') {
+            return 'You have reached the maximum number of requests for your license. You must wait for the returned period of time' + res.substring(17) + 'before running requests again.';
+        }
+        if (res.substring(0,5) == 'ERROR') {
+            return 'An error has occurred. Please refer to the description of the error indicated in the' + res.substring(0,5) + 'value.';
+        }
+    };
+
+    switch (command) {
+        case 'test-module':
+            return 'ok';
+        case 'is-it-phishing':
+            return isPhishing(args.url, args.force, args.smart, args.area, args.timeout);
+        default:
+    }
+
+
+
+
+  type: javascript
+  commands:
+  - name: is-it-phishing
+    arguments:
+    - name: url
+      required: true
+      description: URL to be checked if phishing
+    - name: force
+      description: Set true to analyze URL, or false to check whether URL may cause
+        collateral damage to the end user
+      defaultValue: "false"
+    - name: smart
+      description: Set true to force checks on URLs that may cause collateral damage
+        to the end user, or false to ignore the argument
+      defaultValue: "true"
+    - name: area
+      description: The regional area to force using a proxy
+    - name: timeout
+      description: Timeout in milliseconds. Default value set to 10000, with a minimum
+        value of 1000. Once timeout is reached, TIMEOUT response is returned
+    outputs:
```

Comment:
outputs should document what you are pushing to context. so you first need to push something to context and than document it...